cypress/integration/test.js

```diff
@@ -0,0 +1,6 @@
+describe("user views menus", () => {
+    it('', () => {
+      cy.visit("http://localhost:3000");
+
+    });
+  });
```

Comment:
Please fix formatting

Reply:
BTW, why are you adding this test?

Reply:
Adi requested we set up a mock test to make sure that semaphore would go green, so I set this up just to verify that. In our next pull request this file is removed and replaced by an actual test file. Let me know if you would like me to remove this for this pull request. I may have misunderstood that he just wanted to see it would go green with a test but then want me to remove it maybe?